backend/dal/Services/UserService.cs

```diff
@@ -81,7 +81,7 @@ public PimsUser Activate()
                 this.Logger.LogInformation($"User Activation: key:{key}, email:{email}, username:{username}, first:{givenName}, surname:{surname}");
 
                 var person = new PimsPerson() { Surname = surname, FirstName = givenName };
-                this.Context.PimsPeople.Add(person);
+                this.Context.PimsPerson.Add(person);
```

Comment:
I know you are just renaming an existing service here but I'd question the need to add application "users"; e,g, an MOTI admin or gov employee to the PimsPerson table. Wasn't this table meant to contain just contacts associated with things that can be managed by the app (e.g. tenants of leases etc?). 

What value would it have to add an ADM as a contact for example? 

I know this PR might not be the best place to discuss but would love to hear your thoughts @FuriousLlama  and @devinleighsmith